tests/python/unittest/test_operator.py

```diff
@@ -3496,6 +3496,10 @@ def test_special_functions_using_scipy():
     mathematical_core("gammaln", lambda x: mx.sym.gammaln(x), lambda x: scipy_special.gammaln(x),
                      lambda x: scipy_special.psi(x), 0.5, 0.5)
 
+    # erf
+    mathematical_core("erf", lambda x: mx.sym.erf(x), lambda x: scipy_special.erf(x),
```

Comment:
the `mathematical_core` does not perform gradient checking. so these special functions are not completely tested for their gradient calculation. Is that okay?

Reply:
https://github.com/apache/incubator-mxnet/blob/65736917a63d8d04ee87f653ec2ffcf2be595dd1/tests/python/unittest/test_operator.py#L3480

Reply:
yes, but `npout_grad` is calculated from the `backward_numpy_call` method that we pass to this function.
https://github.com/apache/incubator-mxnet/blob/65736917a63d8d04ee87f653ec2ffcf2be595dd1/tests/python/unittest/test_operator.py#L3473-L3474

with the above statement, the backward pass is getting tested. But that is not the same thing as doing "gradient checking", right? which will test for the gradient of the operator without using the `backward_numpy_call`

Reply:
this is more a question than a review comment.

Reply:
Good question. I think the numerical grad test cases for all mathematical_core ops are missing. Since there are many ops, we'd better track that as a separate task.

Reply:
https://github.com/apache/incubator-mxnet/issues/13254

Reply:
Thanks, I will also update it here - https://github.com/apache/incubator-mxnet/issues/10923